src/player.cpp

```diff
@@ -5137,13 +5137,30 @@ void player::suffer()
                     }
                     if( does_talk ) {
                         add_msg( _( "%1$s says: \"%2$s\"" ), i_name_w, i_talk_w );
+                        done_effect = true;
                     }
-                    done_effect = true;
                 }
-                // Bad feeling
-                if( !done_effect && one_turn_in( 4_hours ) ) {
-                    add_msg( m_warning, _( "You get a bad feeling." ) );
-                    add_morale( MORALE_FEELING_BAD, -50, -150 );
+                // Delusions
+                if( !done_effect && one_turn_in( 8_hours ) ) {
+                    if (rng(1, 20) > 5) { // 75% chance
+                        static const std::vector<std::string> paranoid{ _("You feel like the world is out to get you."),
```

Comment:
```suggestion
                        const std::vector<std::string> paranoid{ _("You feel like the world is out to get you."),
```
`static` means the vector is created once, and never changed. Those the messages it contains are translated once when it's created and are not re-translated when the user changes the game language.

Non-static means the vector is created each time time and the messages are translated each time using the current language.